application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalConstants.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.digital.uec.api.authentication.localstub;
+
+public class LocalConstants {
+
+  private LocalConstants() {}
+
+  public static final String USERNAME = "USERNAME";
```

Comment:
Some of these constants look like they may be used for the real integration (like USERNAME, PASSWORD). Can you please identify and extract these out into a new class called AuthenticationConstants.java